automation/docker/hub-compose-fiab.yml

```diff
@@ -1,7 +1,7 @@
 version: '2'
 services:
   hub:
-    image: selenium/hub
+    image: selenium/node-chrome:3.8.1-erbium
```

Comment:
This shouldn't change. This should remain `selenium/hub`, not `selenium/node-chrome`.

Reply:
DOH